service/history/historyEngine.go

```diff
@@ -158,14 +157,91 @@ func (e *historyEngineImpl) StartWorkflowExecution(startRequest *h.StartWorkflow
 		return nil, &workflow.BadRequestError{Message: "Missing or invalid TaskStartToCloseTimeoutSeconds."}
 	}
 
-	// We generate a new workflow execution run_id on each StartWorkflowExecution call.  This generated run_id is
-	// returned back to the caller as the response to StartWorkflowExecution.
-	runID := uuid.New()
-	workflowExecution := workflow.WorkflowExecution{
-		WorkflowId: common.StringPtr(executionID),
-		RunId:      common.StringPtr(runID),
+	// TODO
```

Comment:
TODO should include what exactly should be done.

Reply:
forgot to remove this TODO.